<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="8" hostname="WTC-5159473-l2" name="Mission_Critical.MC_PI_3" tests="15" failures="7" timestamp="20 May 2019 20:30:13 GMT" time="1866.673" errors="0">
  <testcase name="WFCL_Alliance_Marketing_CodeRequired" time="359.796" classname="Mission_Critical.MC_PI_3">
    <failure type="java.lang.AssertionError" message="bh15245713: Text does not match.  By.tagName: body   L3642210300US101118T153422">
      <![CDATA[java.lang.AssertionError: bh15245713: Text does not match.  By.tagName: body   L3642210300US101118T153422
at org.testng.Assert.fail(Assert.java:96)
at Mission_Critical.MC_PI_3.WFCL_Alliance_Marketing_CodeRequired(MC_PI_3.java:184)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- WFCL_Alliance_Marketing_CodeRequired -->
  <testcase name="WFCL_Alliance_Marketing_CodeRequired" time="317.650" classname="Mission_Critical.MC_PI_3">
    <failure type="java.lang.AssertionError" message="bh15245513: Text does not match.  By.tagName: body   L3642210300US101118T153422">
      <![CDATA[java.lang.AssertionError: bh15245513: Text does not match.  By.tagName: body   L3642210300US101118T153422
at org.testng.Assert.fail(Assert.java:96)
at Mission_Critical.MC_PI_3.WFCL_Alliance_Marketing_CodeRequired(MC_PI_3.java:184)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- WFCL_Alliance_Marketing_CodeRequired -->
  <testcase name="WFCL_Alliance_Marketing_CodeRequired" time="316.554" classname="Mission_Critical.MC_PI_3">
    <failure type="java.lang.AssertionError" message="bh15688813: Text does not match.  By.tagName: body   L3642210300US101118T153422">
      <![CDATA[java.lang.AssertionError: bh15688813: Text does not match.  By.tagName: body   L3642210300US101118T153422
at org.testng.Assert.fail(Assert.java:96)
at Mission_Critical.MC_PI_3.WFCL_Alliance_Marketing_CodeRequired(MC_PI_3.java:184)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- WFCL_Alliance_Marketing_CodeRequired -->
  <testcase name="WFCL_Alliance_Marketing_CodeRequired" time="180.741" classname="Mission_Critical.MC_PI_3">
    <failure type="java.lang.AssertionError" message="bh04174718: Check the status of the AEM page.">
      <![CDATA[java.lang.AssertionError: bh04174718: Check the status of the AEM page.
at org.testng.Assert.fail(Assert.java:96)
at Mission_Critical.MC_PI_3.WFCL_Alliance_Marketing_CodeRequired(MC_PI_3.java:184)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- WFCL_Alliance_Marketing_CodeRequired -->
  <testcase name="WFCL_Alliance_Marketing_CodeRequired" time="193.043" classname="Mission_Critical.MC_PI_3">
    <failure type="java.lang.AssertionError" message="bh15246113: Text does not match.  By.tagName: body   L3642210300US101118T153422">
      <![CDATA[java.lang.AssertionError: bh15246113: Text does not match.  By.tagName: body   L3642210300US101118T153422
at org.testng.Assert.fail(Assert.java:96)
at Mission_Critical.MC_PI_3.WFCL_Alliance_Marketing_CodeRequired(MC_PI_3.java:184)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- WFCL_Alliance_Marketing_CodeRequired -->
  <testcase name="WFCL_Alliance_Marketing_CodeRequired" time="213.720" classname="Mission_Critical.MC_PI_3">
    <failure type="java.lang.AssertionError" message="bh15246213: Text does not match.  By.tagName: body   L3642210300US101118T153422">
      <![CDATA[java.lang.AssertionError: bh15246213: Text does not match.  By.tagName: body   L3642210300US101118T153422
at org.testng.Assert.fail(Assert.java:96)
at Mission_Critical.MC_PI_3.WFCL_Alliance_Marketing_CodeRequired(MC_PI_3.java:184)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- WFCL_Alliance_Marketing_CodeRequired -->
  <testcase name="WFCL_Alliance_Marketing_CodeRequired" time="285.169" classname="Mission_Critical.MC_PI_3">
    <failure type="java.lang.AssertionError" message="bh18018917: Check the status of the AEM page.">
      <![CDATA[java.lang.AssertionError: bh18018917: Check the status of the AEM page.
at org.testng.Assert.fail(Assert.java:96)
at Mission_Critical.MC_PI_3.WFCL_Alliance_Marketing_CodeRequired(MC_PI_3.java:184)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:71)
at org.testng.internal.TestMethodWithDataProviderMethodWorker.call(TestMethodWithDataProviderMethodWorker.java:14)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- WFCL_Alliance_Marketing_CodeRequired -->
  <testcase name="WFCL_Rewards_Registration_APAC_AND_LAC" classname="Mission_Critical.MC_PI_3">
    <skipped/>
  </testcase> <!-- WFCL_Rewards_Registration_APAC_AND_LAC -->
  <testcase name="WFCL_Alliance_Marketing" classname="Mission_Critical.MC_PI_3">
    <skipped/>
  </testcase> <!-- WFCL_Alliance_Marketing -->
  <testcase name="WFCL_Rewards_Login_APAC_AND_LAC" classname="Mission_Critical.MC_PI_3">
    <skipped/>
  </testcase> <!-- WFCL_Rewards_Login_APAC_AND_LAC -->
  <testcase name="WIDM_Registration_Email_Validation" classname="Mission_Critical.MC_PI_3">
    <skipped/>
  </testcase> <!-- WIDM_Registration_Email_Validation -->
  <testcase name="WFCL_Alliance_Marketing_Error_Page_Blank" classname="Mission_Critical.MC_PI_3">
    <skipped/>
  </testcase> <!-- WFCL_Alliance_Marketing_Error_Page_Blank -->
  <testcase name="WFCL_DT_Check" classname="Mission_Critical.MC_PI_3">
    <skipped/>
  </testcase> <!-- WFCL_DT_Check -->
  <testcase name="WFCL_Registration_Email_Validation" classname="Mission_Critical.MC_PI_3">
    <skipped/>
  </testcase> <!-- WFCL_Registration_Email_Validation -->
  <testcase name="WFCL_Alliance_Marketing_Error_Page" classname="Mission_Critical.MC_PI_3">
    <skipped/>
  </testcase> <!-- WFCL_Alliance_Marketing_Error_Page -->
</testsuite> <!-- Mission_Critical.MC_PI_3 -->
